dom.bs

```diff
@@ -1138,9 +1196,15 @@ invoked, must run these steps:
    <var>listener</var>'s <b>capture</b> is true, terminate these substeps (and run them for the next
    <a>event listener</a>).
 
+   <li>If <var>listener</var>'s <b>passive</b> is true, set <var>event</var>'s <a>passive flag</a>.
```

Comment:
This seems wrong. Since before we said this would only happen if all listeners are passive. So this is in fact something that is done by those invoking dispatch.